docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -1,24 +1,28 @@
 ---
-title: Development with local Grafana
+title: Develop with a local environment
 ---
 
-# Development with local Grafana
+# Develop with a local environment
 
-This guide allows you to setup a development environment where you run Grafana and your plugin locally. With this, you will be able to see your changes as you add them.
+Follow the steps in this guide to set up a development environment where you run Grafana and your plugin locally. With this setup, you can see your changes as you add them.
+
+## Before you begin
```

Comment:
The developer guide covers this, we could remove this section.

Reply:
Not totally sure if you're suggesting the removal of the opening sentence or the "before you begin" section. We can't remove the opening as that's required by our guidelines.

Removing the "before you begin" section. However, in general, I think the plugin guide doesn't say enough about prerequisites. Of course that doesn't mean that we need to add prerequisites to the individual topics instead of the introduction though.